### DISCUSSION TITLE
Kibana showing windows_eventlog but not a sysmon module

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10323] [Unsupported Versions] [answered]
Hi, I finally got windows data into security onion. But I dont see sysmon categories?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/8a68f14e-2f48-47d9-9e5a-a0983ef0eedc)

But I do show windows_events?
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/5569263/9919473e-3f13-4448-9809-71782ac0d57b)

are windows_events the same as sysmon maybe? not sure.

thanks for any suggestions or advice

**Comment by dougburks** [comment]
On your Windows host, did you install sysmon and configure winlogbeat to pick up the sysmon log?
https://docs.securityonion.net/en/2.3/sysmon.html
https://docs.securityonion.net/en/2.3/beats.html#winlogbeat

**Reply by iqworks** [reply, marked as answer]
HI Doug thanks for the links! I will review them. I am back on my other project right now, but will be back in a couple weeks.

**Reply by iqworks** [reply]
Hi Doug, I finally see SYSMON in kibana !!!! Those two links above where very helpful. I had to make my VM standalone and I think I needed to enter the host in setup.kibana.
Thanks very much
